src/components/navbar/index.tsx

```diff
@@ -238,12 +244,26 @@ const NavBar: React.FC<NavBarProps> = ({ tokens }) => {
           <Row gutter={[8, 0]}>
             {privilegeLevel !== PrivilegeLevel.NONE ? (
               <Col>
-                <Dropdown overlay={userMenu}>
+                <UserDropdown overlay={userMenu}>
                   <Button>
-                    <UserOutlined />
+                    <UserAvatar
+                      // size="large"
+                      src={
+                        asyncRequestIsComplete(contacts) &&
```

Comment:
We should check if the user has a profile picture, because it can be null. 

`asyncRequestIsComplete(contacts) && contacts.result.mainContact.profilePicture ? (`

Reply:
But I guess it's working given your screenshots..? If so, disregard my comment